CONTRIBUTING.adoc

```diff
@@ -31,18 +31,18 @@ Please ensure the documentation is updated to include any new feature, functiona
 
 You should ensure that your documentation changes include the following:
 
-- Update {uri-changelog}[CHANGELOG] under the 'Unreleased' section. The maintainers will include this to the appropriate version section during the release process for that version
+- Update xref:CHANGELOG.adoc[CHANGELOG] under the 'Unreleased' section. The maintainers will include this to the appropriate version section during the release process for that version
```

Comment:
Perhaps: "Add an entry that explains your fix or enhancement to the list of unreleased items. The maintainers will move your entry to the appropriate version during the release process for that version."